### ISSUE TITLE
Quetion about searching reasults pbt_global.txt

### ISSUE THREAD

**Issue by yichuan9527** [arcelien/pba#25]
Hello, I run the searching algorithm and get the results a pbt_global.txt. 
In the  pbt_global.txt, for example:
`["2", "0", 2, 3, {"aug_policy": "cifar10", "train_size": 4000, "wrn_size": 32, "explore": "cifar10", "data_path": "/home/jinlin3/pba2/datasets/cifar-10-batches-py", "batch_size": 128, "dataset": "cifar10", "no_aug": false, "recompute_dset_stats": false, "test_batch_size": 25, "gradient_clipping_by_global_norm": 5.0, "use_hp_policy": true, "validation_size": 46000, "num_epochs": 200, "hp_policy": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], "weight_decay_rate": 0.0005, "lr": 0.1, "no_cutout": false, "wrn_depth": 40, "model_name": "wrn"}, {"aug_policy": "cifar10", "train_size": 4000, "wrn_size": 32, "explore": "cifar10", "data_path": "/home/jinlin3/pba2/datasets/cifar-10-batches-py", "batch_size": 128, "validation_size": 46000, "no_aug": false, "recompute_dset_stats": false, "wrn_depth": 40, "test_batch_size": 25, "gradient_clipping_by_global_norm": 5.0, "use_hp_policy": true, "dataset": "cifar10", "num_epochs": 200, "hp_policy": [0, 3, 4, 0, 4, 6, 3, 0, 0, 1, 0, 8, 0, 1, 3, 0, 0, 1, 1, 3, 3, 1, 0, 0, 0, 0, 9, 3, 5, 5, 2, 0, 0, 0, 0, 2, 1, 0, 0, 2, 2, 0, 6, 0, 3, 4, 0, 3, 3, 5, 3, 0, 0, 0, 0, 4, 0, 9, 0, 3], "model_name": "wrn", "lr": 0.1, "no_cutout": false, "weight_decay_rate": 0.0005}]
` 
What do "2" and "0" denote? What do 2 and 3 denote ?

**Comment by arcelien** [comment]
Hey, there's a namedtuple that defines each element in the list per line:
https://github.com/arcelien/pba/blob/950397f0afd8562cd05564f11c669ac6567b281d/pba/utils.py#L12-L15

So the elements above correspond to each element in this tuple respectively.

**Comment by yichuan9527** [comment]
Should I use the augmentation strategy of the highest performing child-model to train my model? (For example, if the third child model have highest test accuracy, should I use pbt_policy_3.txt to train mine model ?)

**Comment by yichuan9527** [comment]
I am so sorry. I still can not understand the meaning of 'target_trial_name', 'clone_trial_name', 'target_trial_epochs',  and  'clone_trial_epochs', 'old_config', 'new_config' .

**Comment by arcelien** [comment]
Yes, I'd recommend using the best performing child model to re-train.

The schedule params originate from https://github.com/ray-project/ray/blob/cdc9227f1b7c280465f76639ebc8c28ef7fc332e/python/ray/tune/schedulers/pbt.py#L253-L260

We're logging the trial ids, epoch/iteration numbers, and hyperparameter dict for the original worker and then one we're exploiting to.

**Comment by snownus** [comment]
@arcelien  could you share the codes to visualize the learned policy such as "Probability Hyperparameters Over Time" and "Magnitude Hyperparameters Over Time" in the github homepage?

**Comment by prob1995** [comment]
@snownus  I wrote a .ipynb trying to reproduce the figure in the paper. Maybe you can use this.
https://github.com/prob1995/pba/blob/master/Policy_Visualization.ipynb

**Comment by arcelien** [comment]
@prob1995 wrote a great visualizer you can try. Unfortunately, the original visualization was done in google sheets / charts so there's no code.
